### PR TITLE
fix: importer looks for right key for storage location

### LINKED ISSUE #338: Importer fails on current schema
According to schema, there is not `locations` key in the storage. Just `location`. 

https://github.com/elixir-luxembourg/daisy/blob/77f836a37dc07300ac0250d0b02fc7742230d369/core/importer/datasets_importer.py#L152

https://github.com/elixir-luxembourg/daisy/blob/77f836a37dc07300ac0250d0b02fc7742230d369/core/fixtures/json_schemas/elu-dataset.json#L47

### PR DESCRIPTION
**Pull request by vildead**
Fixes #338

### CODE REVIEW

File: core/importer/datasets_importer.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def process_data_locations(self, dataset, dataset_dict):
                 category = self.process_location_category(storage_location_dict)
 
 
-                location_delimeted = '\n'.join(storage_location_dict['locations'])
+                location_delimeted = '\n'.join(storage_location_dict['location'])
```
Comment (jLebioda) [inline core/importer/datasets_importer.py:152]: If it's going to be a string (and not dict anymore), `'\n'.join()` might be removed

**Review by jLebioda — APPROVED** [review]
are the changes reflected also in the dataset schema?

File: core/importer/datasets_importer.py (hunk 1 above)
Comment (jLebioda) [reply core/importer/datasets_importer.py:152]: but it's no biggie, as string.join(X) works for both strings and iterables

Comment (vildead) [reply core/importer/datasets_importer.py:152]: yes! I wanted to keep that - I am not sure we wont go back to list/dict at some point ;) 
Thanks!

**Pull request merged**